pages/linux/gs.md

```diff
@@ -0,0 +1,27 @@
+# gs
+
+> GhostScript is a PDF and PostScript interpreter.
+
+- To view a file:
+
+`gs -dQUIET -dBATCH {{file.pdf}}`
+
+- Reduce PDF file size to 150 dpi images for reading on a ebook device:
+
+`gs -dNOPAUSE -dQUIET -dBATCH -sDEVICE=pdfwrite -dPDFSETTINGS=/ebook -sOutputFile={{output.pdf}} {{input.pdf}}`
+
+- Convert PDF file to an image:
+
+`gs -dQUIET -dBATCH -dNOPAUSE -sDEVICE=jpeg -r150 -dTextAlphaBits=4 -dGraphicsAlphaBits=4 -dFirstPage=1 -dLastPage=3 -sOutputFile={{output_%d.jpg}} {{input.pdf}}`
```

Comment:
Do these have sane defaults ? Can we not mention them - `-dTextAlphaBits=4 -dGraphicsAlphaBits=4` ?

Also, no need for firstPage and lastPage. They are already mentioned in the next example. Let these convert the entire pdf file.

Reply:
truth to be told I dont know what those defaults are, I always copypaste these 2 options without checking if they are actually neede. I removed them, hopefully they are not needed :-)

Reply:
cargo-culting is in our blood. :)